apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -100,10 +100,19 @@ def _create_empty(
                 else:
                     raise TypeError(f"Unsupported dtype {dtype}")
 
+            extent = create_options.dim_tile(index_column_name)
+            if (
+                dtype == pa.int8()
+                or dtype == pa.uint8()
+                or dtype == np.int8
+                or dtype == np.uint8
+            ):
```

Comment:
Minor drive-by question: can the two first conditions happen? `dtype` here is the result of `util_arrow.tiledb_type_from_arrow_type`, which IIUC can only be a `np.dtype` instance or the string "ascii" or "bytes". So this condition can be simplified to `if isinstance(dtype, np.dtype) and dtype.itemsize == 1:` 
(or perhaps more tersely `if getattr(dtype, "itemsize", 0) == 1:`).

Reply:
Awesome, thanks @gsakkis !